docs/hexa.md

```diff
@@ -0,0 +1,47 @@
+
+# OpenHEXA Toolbox OpenHEXAClient
+
+The OpenHEXAClient module enables users to interact with the OpenHEXA backend using GraphQL syntax. 
+Its primary goal is to simplify communication with OpenHEXA and streamline integration with third-party applications.
+
+* [Installation](#installation)
+* [Example](#example)
+
+
+## [Installation](#)
+
+``` sh
+pip install openhexa.toolbox
+```
+
+## [Example](#)
```

Comment:
Since the client is quite lightweight, I think it would be beneficial to include additional GraphQL query examples.

I'd propose adding the following:

	1.	Getting the details of a given connection
	2.	Browsing datasets and downloading a file

What do you think, @qgerome?

Reply:
Yes, i will add them, thx.